src/Router.js

```diff
@@ -1,24 +1,37 @@
 import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { Outlet } from 'react-router-dom';
+
 import Main from './pages/Main/Main';
 import Nav from './Components/Nav/Nav';
 import Cart from './pages/Cart/Cart';
 import ItemDetail from './pages/ItemDetail/ItemDetail';
 import ItemList from './pages/ItemList/ItemList';
 import Login from './pages/Login/Login';
 import Register from './pages/Register/Register';
+import Footer from './Components/Footer/Footer';
+
+const NavbarLayout = () => (
+  <>
+    <Nav />
+    <Footer />
+
+    <Outlet />
```

Comment:
Outlet 컴포넌트의 위치가 Nav와 Footer 사이에 있어야 할 것 같습니다.
```suggestion
    <Nav />
    <Outlet />
    <Footer />
```